### ISSUE TITLE
Opening .xmind files

### ISSUE THREAD

**Issue by rvalle** [TomasTomecek/fedora_rpm_xmind#5]
After installing on Fedora 24, xmind files are still recognized as ZIP.

Somewhere there must be a extension -> mime type -> application setup.

Still trying to work it out.

Would be nice to have this setup at install

**Comment by TomasTomecek** [comment]
I agree this would be a very nice to have feature. Not sure how it's being done, would have to investigate.

**Comment by TomasTomecek** [comment]
So after a brief internet search it looks like it should work: [application/xmind](https://github.com/TomasTomecek/fedora_rpm_xmind/blob/master/xmind.xml#L3) is [associated](https://github.com/TomasTomecek/fedora_rpm_xmind/blob/master/xmind.desktop#L10) with the application itself.
